articles/tutorials/04_working_with_textures/index.md

```diff
@@ -0,0 +1,364 @@
+
+## Loading a Texture
+
+To load a texture in your game, you can either load it directly from its file (*.png*, *.jpg*, etc) or you can pre-process it using the *content pipeline* and load it using the `ContentManager`.
+
+The *content pipeline* can be used for more than just pre-processing images as textures; you can use it to also load audio, fonts, effects, and 3D models, all of which will be covered in upcoming chapters. It can also be customized to load various other types of file formats, however these extension are not covered in this tutorial.
+
+Let's go over both methods of loading a texture, and then we can discuss the pros and cons of each one. First, we need an image to load. Perform the following:
+
+1. Locate the *Content* directory inside your game project directory
+2. Create a new folder inside the *Content* directory called *images*.
+3. Right-click the following image of the MonoGame logo and save it in the *Content/images* directory you just created with the filename *logo.png*
+
+<figure><img src="./images/logo.png" alt="Figure 4-1: MonoGame Horizontal Logo"><figcaption><p><strong>Figure 4-1: MonoGame Horizontal Logo</strong></p></figcaption></figure>
+
+### Loading From File
+
+First we'll go over the process of loading the texture directly from the file at runtime. Open the *Game1.cs* file in your project and add the following instance member to the class where the `GraphicsDeviceManager` and `SpriteBatch` instance member variables are:
+
+```cs
+private Texture2D _logo;
+```
+
+This adds a new instance member to the class called `_logo` that is of type `Texture2D`. `Texture2D` is the type used to store a reference to 2D image data in MonoGame.
+
+Next, find the `LoadContent` method and add the following after the `_spriteBatch` is instantiated:
+
+```cs
+_logo = Texture2D.FromFile(GraphicsDevice, "Content/images/logo.png");
+```
+
+If you run the game now by press F5, the game will crash with the following exception:
+
+```
+Exception has occurred: CLR/System.IO.DirectoryNotFoundException
+An unhandled exception of type 'System.IO.DirectoryNotFoundException' occurred in System.Private.CoreLib.dll: 'Could not find a part of the path 'C:\dev\MonoGameSnake\MonoGameSnake\bin\Debug\net8.0\Content\images\logo.png'.'
+```
+
+Your path may be different, but you'll get the same exception. This is because even though we added the *logo.png* file, we need to tell the project to copy the file to the project output when a build occurs. To do this, open the *MonoGameSnake.csproj* file and add the following before the closing `</Project>` element at the bottom:
+
+```xml
+<ItemGroup>
+    <Content Include="Content\images\logo.png" CopyToOutputDirectory="PreserveNewest" />
+</ItemGroup>
+```
+

```

Comment:
"If you are using Visual Studio, the same can be achieved by looking for your newly added file in your project view, right click on it/properties and in the Copy to Output Directory field, make sure you select 'copy always'"